From a0568cbcb71a24b3ceeb94b9703b4138c2b3f9ad Mon Sep 17 00:00:00 2001
From: ZhongYiChong <zyc@rock-chips.com>
Date: Fri, 14 Sep 2018 11:07:00 +0800
Subject: [PATCH] arm: dts: rockchip: rk312x: add cma_region reference for CIF

Because rk312x CIF has no IOMMU and it should use CMA memory,
adding the cma_region reference for CIF, so that it could get
the CMA memory from our reserved CMA region other than from
system default CMA region.

Change-Id: I7a52abce75a2a82d3068721ddbb10ed89796ac2a
Signed-off-by: Zhong Yichong <zyc@rock-chips.com>
---
 arch/arm/boot/dts/rk312x-android.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rk312x-android.dtsi b/arch/arm/boot/dts/rk312x-android.dtsi
index 617a20653e5e..745025573b65 100644
--- a/arch/arm/boot/dts/rk312x-android.dtsi
+++ b/arch/arm/boot/dts/rk312x-android.dtsi
@@ -65,6 +65,10 @@
 	};
 };
 
+&cif_new {
+	memory-region = <&cma_region>;
+};
+
 &dfi {
 	status = "okay";
 };
-- 
2.35.3

